examples/lighting-app/efr32/src/AppTask.cpp

```diff
@@ -137,9 +137,17 @@ int AppTask::Init()
     uint32_t setupPinCode;
     std::string QRCode;
 
-    if (GetQRCode(setupPinCode, QRCode, chip::RendezvousInformationFlags::kBLE) == CHIP_NO_ERROR)
+    if (ConnectivityMgr().GetSetupPinCode(setUpPINCode) == CHIP_NO_ERROR)
```

Comment:
I guess you could also call `LogDeviceConfig` here and in other samples which use `PrintQRCode` functions. I know it's not our responsibility, but it seems straightforward and we would keep all platforms aligned. ;)

Reply:
1. I tried to fix all platforms (not all of them are using QRCodeUtil module and some have own internal methods), that were using GetQRCode and could be affected by API changes and it seemed to me that EFR32 is the only one. Now I see that QPG6100 is also using PrintQRCode, but I don't see anything more beside those two.
2. I got your point and I can do so, but my goal was to not modify anything for other platforms and stay with the same logs as before to not touch others stuff without asking. On the other hand I think that platform owners will not be upset because of some additional logs.